index.js

```diff
@@ -2,8 +2,7 @@
 
 module.exports = YahooFantasy;
 
-var OAuth = require('oauth').OAuth,
-  https = require('https'),
+var request = require('request'),
```

Comment:
I'm playing around with OAuth2, something similar to what you did. I do like it, and I do think it makes sense to remove the refresh token and just fire an event for the user to listen for which will allow them to handle the refresh themselves.

However, the issue I'm having with OAuth2 is that I can no longer query the public data without the user authenticating? That is, I cannot even query game.meta without being authenticated. Is there any way to provide Client credentials (Consumer Key / Consumer Secret for the app) in order to query Yahoo! ?